ReactiveCocoaTests/Swift/SignalSpec.swift

```diff
@@ -1566,60 +1569,88 @@ class SignalSpec: QuickSpec {
 				combinedValues = nil
 				completed = false
 				
-				let (signalA, baseSinkA) = Signal<Int, NoError>.pipe()
-				let (signalB, baseSinkB) = Signal<Int, NoError>.pipe()
-				let (signalC, baseSinkC) = Signal<Int, NoError>.pipe()
+				let (baseSignalA, baseSinkA) = Signal<Int, NoError>.pipe()
+				let (baseSignalB, baseSinkB) = Signal<Int, NoError>.pipe()
+				let (baseSignalC, baseSinkC) = Signal<Int, NoError>.pipe()
+				
+				signalA = baseSignalA
+				signalB = baseSignalB
+				signalC = baseSignalC
 				
 				sinkA = baseSinkA
 				sinkB = baseSinkB
 				sinkC = baseSinkC
-				
-				combineLatest(signalA, signalB, signalC)
-				|> observe(next: {
-						combinedValues = [$0, $1, $2]
-					}, completed: {
-						completed = true
-					})
 			}
 			
-			it("should forward the latest values from all inputs"){
-				expect(combinedValues).to(beNil())
+			sharedExamples("combineLatest examples") {
```

Comment:
Can you please use a constant string for this, so we can refer to the constant by name instead of duplicating the string contents below?